js/build.js

```diff
@@ -0,0 +1,344 @@
+                  ? file
+                  : file.url;
+
+                return Fliplet.Media.getIdFromRemoteUrl(url);
+              });
+
+              // TODO check how to escape this call
+              return Fliplet.Media.Files.getAll({
+                files: fileIDs,
+                fields: ['name', 'url', 'metadata', 'createdAt']
+              }).then(function (files) {
+                var filesInfo = files.map(function (file) {
+                  const extension = file.name.split('.').pop().toLowerCase();
+                  var type = '';
+                  if (extension === 'jpg' || extension === 'jpeg' || extension === 'png' || extension === 'gif') {
+                    type = 'image';
+                  } else if (extension === 'mp4' || extension === 'avi' || extension === 'mkv' || extension === 'mov') {
+                    type = 'video';
+                  } else {
+                    type = 'file';
+                  }
+                  return {
+                    type,
+                    name: file.name,
+                    size: file.metadata.size,
+                    uploaded: file.createdAt,
+                    url: file.url
+                  };
+                }).sort(sortFilesByName);
+                if (type == 'Image') {
+                  var data = {
+                    images: filesInfo.map(el => {
+                      return {
+                        // title: el.name,
+                        url: Fliplet.Media.authenticate(el.url)
+                      }
+                    }),
+                    options: {
+                      index: 0,
+                      errorMsg: 'The photo cannot be loaded'
+                    }
+                  };
+                  var images = data.images.map((el, index) => {
+                    return `<div class="image-item-container" data-index="${index}">
+                              <img src="${el.url}" />
+                            </div>`
+                  })
+                  $(document).find('[data-helper="file-list"]').append(`
+                   <div class="image-container">
+                   ${images.join('')}
+                   <div>
+                  `);
+
+                  $(document).find('.image-item-container').off('click');
+                  $(document).find('.image-item-container').on('click', function () {
+                    data.options.index = Number($(this).attr('data-index'));
+                    Fliplet.Navigate.previewImages(data);
+                  })
+                } else {
+                  var str = '';
+                  fileItems = [];
+                  filesInfo.forEach(el => {
+                    fileItems.push(`<div class="file-container-item" data-link="${encodeURIComponent(el.url)}">
+                    <div>
+                      <p>${el.name}</p>
+                      <p>Uploaded: ${moment(el.uploaded).format('MMM D, YYYY')} - ${convertBytesToLargerUnits(el.size)}</p>
```

Comment:
Use our localisation framework and see **Locale aware formats** in https://momentjs.com/docs/

```suggestion
                      <p>Uploaded: ${TD(el.uploaded, 'LL')} - ${convertBytesToLargerUnits(el.size)}</p>
```